data/campaigns/World_Conquest/scenarios/WC_II_scenario.cfg

```diff
@@ -101,6 +101,75 @@ _ "World Conquest 3p" #enddef
         # Expert is a string that was supposed to be in wesnoth-wc but accidentally got added to wesnoth-units instead
         #textdomain wesnoth-units
         {WC2_CAMPAIGN_DIFFICULTY NIGHTMARE {WC2_NIGHTMARE_DIFFICULTY} {STR_NIGHTMARE} _"Expert" 9 7 1 0 no 20}
+
+        [about]
+            title = _ "Campaign Design"
+            [entry]
+                name = "tekelili"
+            [/entry]
+        [/about]
+
+        [about]
+            title = _ "Prose, Grammatical and Text Assistance"
+            [entry]
+                name = "TL"
+            [/entry]
+            [entry]
+                name = "Natasiel"
+            [/entry]
+            [entry]
+                name = "Rigor"
+            [/entry]
+            [entry]
+                name = "tekelili"
+            [/entry]
+        [/about]
+
+        [about]
+            title = _ "Minor Code Contributors"
+            [entry]
+                name =  "ezysquire"
+            [/entry]
+            [entry]
+                name =  "tsr"
+            [/entry]
+        [/about]
+
+        [about]
+            title = _ "Playtesting"
+            [entry]
+                name =  "Bear"
+            [/entry]
+            [entry]
+                name =  "Honor"
+            [/entry]
+            [entry]
+                name =  "jb"
+            [/entry]
+            [entry]
+                name =  "paso"
+            [/entry]
+            [entry]
+                name =  "tekelili"
+            [/entry]
+        [/about]
+
+        [about]
+            title = _ "Campaign Maintenance"
+            [entry]
+                name = "gfgtdf"
+                comment= "former maintainer"
+            [/entry]
+            [entry]
+                name =  "Celtic_Minstrel"
```

Comment:
I prefer to use a space rather than an underscore.

Reply:
Done